cogboard-webapp/src/components/Widget/index.js

```diff
@@ -138,7 +140,10 @@ const Widget = ({ id, index }) => {
           }
         />
         <StyledCardContent>
-          {!disabled ? <WidgetContent id={id} type={type} content={content} /> : 'Disabled'}
+          {content && content.errorMessage && content.errorMessage !== ''
```

Comment:
I suggest moving this to helper function called renderCardContent taking content and disable as arguments. This component is getting more and more bloated. I plan to do some serious refactor.
I also think content.errorMessage !== '' is redundant to content.errorMessage which will coerce to false if it is an empty string.

Reply:
done